src/services/consent/model.js

```diff
@@ -1,15 +1,14 @@
-'use strict';
-
 // model.js - A mongoose model
 //
 // See http://mongoosejs.com/docs/models.html
 // for more of what you can do here.
 
 const mongoose = require('mongoose');
+const mongooseHistory = require('mongoose-history');
 
 const { Schema } = mongoose;
 
-const consentForm = ['analog', 'digital'];
+const consentForm = ['analog', 'digital', 'update'];
```

Comment:
Im not sure what the semantic of the "update" form would be... isnt that digital too?

Reply:
jap, but the difference is that we have not an checked checkbox here but only informed the user